db/migrate/20230720125622_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Creates the posts table in the database
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :LikesCounter
+      t.string :CommentsCounter
```

Comment:
- [ ] @Adams-Ghub Great work on your migration file, however, this field should be of type integer and not of type string.

Reply:
well resolved! ✔️